Empire/Empire.cpp

```diff
@@ -2924,6 +2993,19 @@ void Empire::CheckProductionProgress() {
             continue;
         }
 
+        // Add the to the imperial stockpile
+        if (elem.item.build_type == BT_STOCKPILE) {
+            /*Meter*ind_meter = build_location->GetMeter(METER_INDUSTRY);
+            if (!meter)
+                continue;
+            float cur_ind_meter = meter->Current();
+            float new_ind_meter = cur_ind_meter - elem.allocated_pp; // * elem.blocksize;
+            ind_meter->SetCurrent(new_ind_meter);
+            ind_meter->BackPropagate();*/
+            Meter* meter = this->GetMeter("METER_STOCKPILE");
+            meter->SetCurrent(meter->Current() + elem.allocated_pp);
```

Comment:
Probably shouldn't set the meter directly here, but rather update how `ProductionQueue::m_expected_new_stockpile_amount` is determined.